apps/desktop/src/renderer/routes/_authenticated/settings/project/$projectId/cloud/secrets/components/SecretsSettings/SecretsSettings.tsx

```
@@ -0,0 +1,184 @@
+
+	const suggestedMatch = useMemo(() => {
+		if (!project || project.neonProjectId || !cloudProjects) return null;
+		const repoName = project.mainRepoPath.split("/").pop();
+		if (!repoName || !project.githubOwner) return null;
+		return cloudProjects.find(
+			(cloud) =>
+				cloud.repoOwner === project.githubOwner && cloud.repoName === repoName,
+		);
+	}, [project, cloudProjects]);
+
+	useEffect(() => {
```

Biome (2.3.14)

[error] This hook specifies a dependency more specific than its captures: linkToNeon.mutate

This capture is more generic than...

...this dependency.


(lint/correctness/useExhaustiveDependencies)